frontend/src/pages/add-quiz-page.tsx

```diff
@@ -0,0 +1,138 @@
+import { Dispatch, SetStateAction, useState } from "react";
+import { Link } from "react-router-dom";
+
+type Question = {
+    id: number,
+    word: string,
+    options: string[],
+    type: string
+}
+
+type Quiz = {
```

Comment:
This type is already exported from `src/types/question.tsx`, we can import from there to stay consistent.

Reply:
Same thing I said for Question type above.

For Quiz type I think we should have the following fields:

{
  id: number,
  name: string,
  description: string,
  imageUrl: string,
  questions: Question[]
}

What do you think?

Reply:
I've added these fields in #298, we can merge this and integrate this page to it with that PR.